db/schema.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  pgTable,
+  bigserial,
+  integer,
+  varchar,
+  timestamp,
+  uniqueIndex,
+} from "drizzle-orm/pg-core";
+
+export const imageTable = pgTable("image", {
+  id: bigserial("id", { mode: "number" }).primaryKey(),
+  prompt: varchar("prompt").notNull(),
+  printifyImageId: varchar("printify_image_id").notNull(),
+  printifyImageUrl: varchar("printify_image_url").notNull(),
+  printifyProductId: varchar("printify_product_id").notNull(),
+  createdAt: timestamp("created_at", { mode: "date" }).defaultNow().notNull(),
+});
+
+export const orderTable = pgTable(
+  "order",
+  {
+    id: bigserial("id", { mode: "number" }).primaryKey(),
+    printifyOrderId: varchar("printify_order_id"),
+    printifyProductId: varchar("printify_product_id").notNull(),
+    printifyVariantId: varchar("printify_variant_id").notNull(),
+    stripeCustomerId: varchar("stripe_customer_id"),
+    quantity: integer("quantity").notNull(),
+    status: varchar("status").notNull().default("payment_pending"),
+    createdAt: timestamp("created_at", { mode: "date" }).defaultNow().notNull(),
+  },
+  (orders) => {
+    return {
+      uniqueOrderIdx: uniqueIndex("unique_order_idx").on(
+        orders.printifyOrderId,
+      ),
+      uniqueCustomerIdx: uniqueIndex("unique_customer_idx").on(
+        orders.stripeCustomerId,
+        orders.printifyProductId,
+      ),
```

Comment:
so a customer can't order the same printify product twice? or can they?

stripe customer id for a given email is always the same

but product_id changes per prompt?